src/test/java/com/amazon/opendistroforelasticsearch/security/test/AbstractSecurityUnitTest.java

```diff
@@ -253,12 +253,12 @@ public Settings get(int i) {
         };
     }
 
-    protected NodeSettingsSupplier minimumSecuritySettingsSslOnly(Settings other, boolean hasCustomTransportSettings) {
+    protected NodeSettingsSupplier minimumSecuritySettingsSslOnly(Settings other) {
 
         return new NodeSettingsSupplier() {
             @Override
             public Settings get(int i) {
-                return minimumSecuritySettingsBuilder(i, true, hasCustomTransportSettings).put(other).build();
+                return minimumSecuritySettingsBuilder(i, true, hasCustomTransportSettings(other)).put(other).build();
```

Comment:
Change `minimumSecuritySettingsBuilder()` to take `Settings other` instead of `boolean hasCustomTransportSettings` and call `hasCustomTransportSettings()` from `minimumSecuritySettingsBuilder()`. Call to `put(other)` should also be part of `minimumSecuritySettingsBuilder()`.